octobot_commons/pretty_printer.py

```diff
@@ -186,11 +210,10 @@ def pretty_print_dict(dict_content, default="0", markdown=False) -> str:
     _, _, code = get_markers(markdown)
     if dict_content:
         result_str = octobot_commons.DICT_BULLET_TOKEN_STR
-        return (
-            f"{result_str}{code}"
-            f"{octobot_commons.DICT_BULLET_TOKEN_STR.join(f'{value} {key}' for key, value in dict_content.items())}"
-            f"{code}"
+        pretty_dict = octobot_commons.DICT_BULLET_TOKEN_STR.join(
+            f"{value} {key}" for key, value in dict_content.items()
         )
+        return f"{result_str}{code}" f"{pretty_dict}" f"{code}"
```

Comment:
Why this change?

Reply:
Oh this slipped in for debugging I think, I removed it